devices/sylvania.js

```diff
@@ -189,4 +189,12 @@ module.exports = [
         extend: extend.ledvance.light_onoff_brightness(),
         ota: ota.ledvance,
     },
+    {
+        zigbeeModel: ['A19 G2 RGBW'],
+        model: '75564',
+        vendor: 'Sylvania',
+        description: 'Smart+ Adjustable White and Full Color Bulb A19',
+        extend: extend.light_onoff_brightness_colortemp_color(),
```

Comment:
Please add the color temperature range as described on https://www.zigbee2mqtt.io/advanced/support-new-devices/01_support_new_devices.html#_3-adding-converter-s-for-your-device

Reply:
Added colortemp... I think... again my apologies for my noob-ness